openai/src/commonMain/kotlin/com/xebia/functional/xef/auto/llm/openai/OpenAI.kt

```diff
@@ -44,6 +48,8 @@ class OpenAI(internal val token: String) {
         ?: throw AIError.Env.OpenAI(nonEmptyListOf("missing OPENAI_TOKEN env var"))
     }
 
+    // TODO: Handle the lifetime of these values!
+
     @JvmField val DEFAULT = OpenAI(openAITokenFromEnv())
```

Comment:
Right now I'm trying to figure out how the lifetime of a companion object is handled. All `OpenAIModel` values defined below are tied to the lifetime of this `DEFAULT` instance of `OpenAI`, so they will correctly be closed when `DEFAULT` is closed (and, thus, closing the OpenAI `client`). However, `DEFAULT` is instantiated in the companion object of the class and I have to make sure that it's closed somewhere (or make sure that the `OpenAIModel` instances it defines are not initialised unless actually being used

Reply:
The existence of an instance to be used by default implies that it's lifetime is the whole's application's lifetime. That is, we either have a default instance built at the beginning of the application, or we can make sure its lifetime is as short as needed. So I guess that this PR should end here, with future work dealing with how we can provide users with default AI models to use.